### DISCUSSION TITLE
en/guestbook/

### DISCUSSION THREAD

**Discussion by giscus** [guangzhengli/blog-comments#17] [Announcements]
# en/guestbook/

Ladder@Guestbook - GuangzhengLi

https://guangzhengli.com/en/guestbook/

**Comment by think-crow** [comment]
一个很好的建议：文章详细内容->标题摘要放在侧面会更好看一些；
像这个博客一样：[我们不谈论 Quarto](https://www.apreshill.com/blog/2022-04-we-dont-talk-about-quarto/)。
感觉这样会更简洁方便！

**Comment by zheqiaochen** [comment]
很好看的网站
